src/pages/index.js

```diff
@@ -14,6 +16,48 @@ const config = {
   errorClass: "modal__error_visible",
 };
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "96d8d88b-e023-4fc5-b12a-b613f7a6a6e2",
+    "Content-Type": "application/json",
+  },
+});
+
+let section; // undefined
+
+api
+  .getInitialCards()
+  .then((cards) => {
+    section = new Section(
+      {
+        items: cards,
+        renderer: (cardData) => {
+          const card = getCardElement(cardData);
+          // create a new card with cardData
+          section.addItem(card);
+        },
+      },
+      selector
+    );
+    section.renderItems();
+    // render initial cards here
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+const userInfo = new UserInfo(
+  "#profile-name",
+  "#profile-description",
+  ".profile__image"
+);
+
+api.getUserInfo().then((userData) => {
+  userInfo.setUserInfo(userData);
+  userInfo.setAvatar(userData.avatar);
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**